src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -72,10 +72,13 @@ namespace OpenMS
     defaults_.setValidStrings("write_convex_hull", {"true","false"});
     defaults_.setValue("spectrum_addition_method", "simple", "For spectrum addition, either use simple concatenation or use peak resampling", {"advanced"});
     defaults_.setValidStrings("spectrum_addition_method", {"simple", "resample"});
-    defaults_.setValue("add_up_spectra", 1, "Add up spectra around the peak apex (needs to be a non-even integer)", {"advanced"});
-    defaults_.setMinInt("add_up_spectra", 1);
+    defaults_.setValue("add_up_spectra", 1, "Add up spectra around the peak apex (needs to be a non-even integer). If set to -1, it will automatically select the number of spectra to add up around the peak apex.", {"advanced"});
```

Comment:
```suggestion
    defaults_.setValue("add_up_spectra", 1, "Add up spectra around the peak apex (needs to be a non-even integer). If set to -1, it will automatically select the number of spectra to add up around the peak apex according to use_percent_peak_width.", {"advanced"});
```